dashboard/db/schema.rb

```diff
@@ -283,6 +283,7 @@
     t.string   "type",                     limit: 255
     t.string   "md5",                      limit: 255
     t.boolean  "published",                              default: false, null: false
+    t.string   "notes",                    limit: 255
```

Comment:
how long notes do people want to add? Should it be a text field so it can be longer than 255 char?